### ISSUE TITLE
Doesn't work in Rails 3

### ISSUE THREAD

**Issue by lassebunk** [ncr/background-fu#4]
As you might know, the plugin doesn't work in Rails 3. It would be nice to get this working as this plugin appears to be very good. Thanks in advance.

**Comment by ncr** [comment]
Hey, I don't maintain this code anymore, had a guy who wanted to update it to Rails 3 but he disappeared in the middle of doing this. I personally use delayed_job nowdays. It has the same features (plus a few others). I will of course accept any pull request if you send me any :)

**Comment by lassebunk** [comment]
Yeah I also think I'll be using delayed_job instead. What I like about your plugin was the convention to use workers in the lib directory. I think it's a bit fuzzy how to do this in delayed_job but I'll just have to look into it :-)

**Comment by ccastillop** [comment]
B-fu has also some "bonus features" like stop and delete job and a nice admin to monitoring progress that DJ doesn't have or that is very difficult to implement. I am trying to monitoring/stop/pause a work with DJ...
Doing a fork.. may be can trasnlate to rails 3
